metrics/metrics.go

```diff
@@ -9,7 +9,7 @@ import (
 	"go.opencensus.io/tag"
 )
 
-var defaultMillisecondsDistribution = view.Distribution(0.01, 0.05, 0.1, 0.3, 0.6, 0.8, 1, 2, 3, 4, 5, 6, 8, 10, 13, 16, 20, 25, 30, 40, 50, 65, 80, 100, 130, 160, 200, 250, 300, 400, 500, 650, 800, 1000, 2000, 5000, 10000, 20000, 30000, 50000, 100000, 200000, 500000, 1000000, 2000000, 5000000, 10000000, 10000000)
```

Comment:
@birdychang I assume this change is the primary motivation of the PR?

Reply:
yes